packages/editor/src/components/list-view-sidebar/style.scss

```
@@ -22,11 +22,17 @@
 	// This allows items in the list view to have equidistant padding left and right
 	// right up until a scrollbar is present.
 	scrollbar-gutter: auto;
+}
 
-	// The table cells use an extra pixels of space left and right. We compensate for that here.
+.editor-list-view-sidebar__list-view-panel-content {
```

Let's please keep the comment.

```suggestion
// The table cells use an extra pixels of space left and right. We compensate for that here.
.editor-list-view-sidebar__list-view-panel-content {
```